docs/devsite-help/support.md

```diff
@@ -0,0 +1,223 @@
+# Support
+
+The vast majority of the code in the Google Cloud Libraries is autogenerated. If
+you're running into problems, it *probably* isn't in that code (although that's
+always a possibility). The problems we see being reported usually fall into a
+few different buckets, described below.
+
+In general, after reading the advice below, if you have a support contract with
```

Comment:
I would move this to the "Reporting a Problem" section.

Reply:
Done, and reworded a bit. See what you think.